generators/server/templates/dotnetcore/src/Project/Configuration/MvcStartup.cs.ejs

```diff
@@ -59,13 +59,19 @@ namespace <%= namespace %>.Configuration
         public static IApplicationBuilder UseApplicationWeb(this IApplicationBuilder app, IHostEnvironment env)
         {
             app.UseDefaultFiles();
+            <%_ if (clientFramework === "Blazor") { _%>
+            app.UseBlazorFrameworkFiles();
```

Comment:
@nicolas63 here blazor configuration code  would not compile if the package reference is missing
Now I see that I forgot to add IncludeBlazor condition here. I will fix it soon

But the package "Microsoft.AspNetCore.Components.WebAssembly.Server" will still be needed for this code to compile. 

Again, I think it is a very small cost and only for the backend project.

Reply:
I also noticed a missing check for INCLUDE_BLAZOR here and pushed new changes